cares_reinforcement_learning/algorithm/PER_TD3.py

```diff
@@ -0,0 +1,138 @@
+"""
```

Comment:
This is essentially a copy and paste of TD3 with a small change on the memory portion - is there a way to modify the memory interface to generic this across other approaches that would use PER. We want to avoid making a PER version of all approaches just for the calculation of the PER data.